tests/units/models/test_construct.py

```diff
@@ -101,15 +105,52 @@ def _make_ferminets():
     slog_psis = []
     # No need for combinatorial testing over these flags; just make sure Ferminet is
     # tested with and without cyclic spins, and with each different determinant_fn_mode.
-    for cyclic_spins, use_det_resnet, determinant_fn_mode, full_det in [
-        (False, False, models.construct.DeterminantFnMode.SIGN_COVARIANCE, False),
-        (False, False, models.construct.DeterminantFnMode.SIGN_COVARIANCE, True),
-        (True, True, models.construct.DeterminantFnMode.SIGN_COVARIANCE, False),
-        (False, True, models.construct.DeterminantFnMode.PARALLEL_EVEN, False),
-        (True, True, models.construct.DeterminantFnMode.PAIRWISE_EVEN, False),
+    for (
+        cyclic_spins,
+        use_det_resnet,
+        determinant_fn_mode,
+        full_det,
+        num_heads,
+        use_transformer,
+    ) in [
+        (
+            False,
+            False,
+            models.construct.DeterminantFnMode.SIGN_COVARIANCE,
+            False,
+            1,
+            False,
+        ),
+        (
+            False,
+            False,
+            models.construct.DeterminantFnMode.SIGN_COVARIANCE,
+            True,
+            1,
+            False,
+        ),
+        (
+            True,
+            True,
+            models.construct.DeterminantFnMode.SIGN_COVARIANCE,
+            False,
+            1,
+            False,
+        ),
+        (
+            False,
+            True,
+            models.construct.DeterminantFnMode.PARALLEL_EVEN,
+            False,
+            1,
+            False,
+        ),
+        (True, True, models.construct.DeterminantFnMode.PAIRWISE_EVEN, False, 1, False),
```

Comment:
Can you modify/add to these tests so that we're testing the use_transformer case with both num_heads=1 and num_heads>1?

Reply:
i see, in this file we haven't enable the transformer test